compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -270,9 +270,12 @@ void addGPUMatmulTensorCorePassPipeline(OpPassManager &pm,
   nestedModulePM.addPass(createCanonicalizerPass());
   nestedModulePM.addPass(createCSEPass());
 
+  PipeliningSchedulingStrategy schedule =
```

Comment:
This way of setting up passes will make it impossible to get good reproducers.... We are not where close to supporting that anyway on the different backends, but just pointing it out that these kinds of patterns are better avoided.